oauth/scopes.py

```diff
@@ -0,0 +1,90 @@
+CREATE = "create"
```

Comment:
Seems like this entire file needs to be a configurable thing that library depends on, not defining the scopes itself.

Reply:
As a first step I just wanted to enable a fully portable, complete OAuth scheme. I think it adds unnecessary complexity to allow consumers to pick and choose what parts of the scheme they want, but that's definitely a change worth making once this is merged.

Reply:
Well, the set of scopes you have listed here seem very application-dependent (e.g. privacy requests, policy rules).

Are you saying that fideslib should define the superset of all scopes used across all fides apps?

Reply:
All I'm saying is that as a first pass, the lib should give you an all-or-nothing OAuth scheme OOB. Enabling consumers to configure a subset of the API routes/OAuth scopes that they import is a worthy change, but I'd prefer to make it in a separate PR.